x-pack/plugins/apm/public/components/routing/templates/apm_service_template/index.tsx

```diff
@@ -265,9 +266,18 @@ function useTabs({ selectedTab }: { selectedTab: Tab['key'] }) {
         path: { serviceName },
         query,
       }),
-      label: i18n.translate('xpack.apm.home.infraTabLabel', {
-        defaultMessage: 'Infrastructure',
-      }),
+      label: (
+        <EuiFlexGroup gutterSize="xs">
+          <EuiFlexItem>
+            {i18n.translate('xpack.apm.home.infraTabLabel', {
+              defaultMessage: 'Infrastructure',
+            })}
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <BetaBadge icon="beaker" />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      ),
```

Comment:
that would be great because the icon will be aligned with the title.

Reply:
Yes, will do!

Reply:
Hi `append` prop is not working for some reason, I asked eui team about it.

Reply:
I checked with @kpatticha, it's possible that because we use a wrapper the prop won't work https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/components/shared/page_template/page_template.tsx

I leave it like this